templates/job-dashboard.php

```diff
@@ -23,7 +23,7 @@
 	exit; // Exit if accessed directly.
 }
 
-$submission_limit			= get_option( 'job_manager_submission_limit' );
+$submission_limit			= ! empty( get_option( 'job_manager_submission_limit' ) ) ? absint( get_option( 'job_manager_submission_limit' ) ) : false;
```

Comment:
`empty(0)` and `empty('0')` is also `true`. I think we need `'' !== get_option( 'job_manager_submission_limit', '' )` to test if it doesn't have any value set.

The `if` below checks for `'false'` (string) instead of the `false` (bool) set here.

Reply:
Fixed in https://github.com/Automattic/WP-Job-Manager/pull/2362/commits/a8d3b34116c82318f4cdefea0633f6aa0a8c2138